31-red-kangaroo/technical_specifications.md

```diff
@@ -0,0 +1,44 @@
+Angular/Vue for Data Portal (Red Kangaroo)
+**Epic Type:** Exploratory Epic
+
+Epic planning and implementation follows the
+[Epic Planning and Marathon SOP](https://docs.ghga-dev.de/main/sops/sop001_epic_planning.html)
+
+## Principal Components of Exploration:
+
+This epic seeks to explore the use of Angular and Vue for a refactoring of the Data Portal (party due to the deprecation of Create-React-App) by implementing some basic Metadata Catalog(ue) functionality on both frameworks and deciding on a potential framework for future refactoring.
+
+### Questions to Answer
+- Are any of the two options suitable for the development of the data portal?
+- If so, which framework is best suited for the development of the data portal?
+- Which framework's data handling system is best suited for the requirements of the data portal?
+- Are Vue's advantages (e.g. shallower learning curve, less restrictive framework) enough of a positive to counter the lack of previous experience with it in the team?
+
+## Material and Resources:
+
+Materials to be studied (e.g. articles, book chapters, youtube videos, etc.) and any (external) persons/experts to talk to:
+
+[- Learning material defined here](https://wiki.verbis.dkfz.de/pages/viewpage.action?pageId=266600453)
+
+## User Journeys (Optional)
+
+This epic covers the following user journeys:
+
+- The user views a list of all datasets
+- The user filters the list of datasets based on facets provided by the API
+- The user can see the summary of the details of the datasets the user selects.
+- The user can see the full details of the datasets the user selects
+- The user views a global summary of the entire metadata database
```

Comment:
I would think only the first two points are required to get a good idea. However, would like to hear @Cito 's opinion on that?

Reply:
Yes, let's keep it simple. The main point is to exercise how data is fetched from the backend and shown. It suffices to do it once.

What could be added is the form for submitting an access requests, in order to see how form handling is supported by the frameworks.

Reply:
Changed :)